lib/system/gc_regions.nim

```diff
@@ -8,23 +8,7 @@
 
 # "Stack GC" for embedded devices or ultra performance requirements.
 
-when defined(nimphpext):
```

Comment:
This section is for writing PHP extensions with native Nim and should be kept.

Reply:
Ok keeping it.